### ISSUE TITLE
Build error on Linux: libbinaryen.a

### ISSUE THREAD

**Issue by lrettig** [ewasm/hera#431]
Getting the following error when I run `cmake --build`:

```
gmake[2]: *** No rule to make target `deps/lib/libbinaryen.a', needed by `src/libhera.so'.  Stop.
gmake[1]: *** [src/CMakeFiles/hera.dir/all] Error 2
gmake: *** [all] Error 2
```

Manually copying `libbinaryen.a` from `deps/lib64` seems to have worked.

Hera commit is https://github.com/ewasm/hera/commits/753dafc4f6e26d47fbbf3a119321689ca785b1b1. I'm running vanilla Amazon Linux:

```
> uname -a
Linux ip-172-31-69-70.ec2.internal 4.14.62-70.117.amzn2.x86_64 #1 SMP Fri Aug 10 20:14:53 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
> cmake --version
cmake version 3.12.2
```

Note that the same error (and workaround) was reported here: https://github.com/moneroexamples/compile-cpp-ethereum-on-arch/issues/3

**Comment by jakelang** [comment]
note that this also sometimes happens on aleth build with `libbinaryen.a`
and `libmpir.a`. I am guessing that binaryen's cmake package may have an
issue.

**Comment by chfast** [comment]
What a fancy distribution do you have? I will fix it.

**Comment by axic** [comment]
It seems to be AWS.

@chfast any luck finding the issue?

@lrettig how did you workaround this?

**Comment by chfast** [comment]
Can you test the fix in #437 ?
